Makefile

```diff
@@ -55,14 +57,23 @@ clone-%: %/.git
 	@(cd $*; git remote rename origin submariner)
 	@(cd $*; git remote add axon $(AXON_NET_GH)/$*.git)
 
-git-fetch-latest:	## Fetch latest repositories from upstream, does *not* rebase
+git-fetch-latest:	## Fetch latest tip from upstream repositories, does *not* rebase
 git-fetch-latest: fetch-latest-admiral fetch-latest-cloud-prepare fetch-latest-lighthouse
 git-fetch-latest: fetch-latest-submariner fetch-latest-submariner-operator fetch-latest-shipyard
 
 fetch-latest-%: clone-%
 	@echo -- $@ --
 	@(cd $*; git fetch submariner devel)
 
+
+git-stable:	## Update repositories to last set tag
+git-stable: stable-admiral stable-cloud-prepare stable-lighthouse
+git-stable: stable-submariner stable-submariner-operator stable-shipyard
+
+stable-%: fetch-latest-%
+	@echo -- moving $@ to latest tag --
```

Comment:
I think we should stick to the `@echo -- $@ --` convention, this makes easier to find which target currently runs